api/prisma/seed.js

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable no-console */
+const { PrismaClient } = require('@prisma/client')
+const dotenv = require('dotenv')
+const countries = require('./countries')
+
+dotenv.config()
+const db = new PrismaClient()
+
+/**
+ *
+ * @template T
+ * @param {Array<T>} array
+ * @param {(item: T) => void} callback
```

Comment:
jsdoc adds types to javascript: https://medium.com/@antonkrinitsyn/jsdoc-generic-types-typescript-db213cf48640